### ISSUE TITLE
Error in option parsing: Error in --lang: Error in option parsing: unknown input language `sygus'

### ISSUE THREAD

**Issue by peleghila** [CVC4/CVC4#4778]
Upgrading to version 1.8, `--lang sygus` returns the error 

```>cvc4-1.8-win64-opt.exe --lang sygus
(error "Error in option parsing: Error in --lang: Error in option parsing: unknown input language `sygus'
Try --language help")
```

despite the fact that `sygus` is listed under `--lang help` and appears to be supported in src/options/language.cpp. (As an aside, `--language help`, the recommended option, is invalid.)

**CVC4 version/commit**: cvc4-1.8-win64-opt.exe downloaded from site (calling `--version` returns `1.9`)
**Operating system**: Windows 10

**Comment by nafur** [comment]
Thanks for the report. About a month ago, support for sygus **version 1** has been removed from CVC4 (see #4651 for details), which was available via `--lang sygus1` at this point.
Now, we only support sygus2 via `--lang sygus2`. I think we currently do not support `--lang sygus` to avoid confusion concerning these different versions.

I hope sygus2 works for you, please reopen this ticket if it does not.

**Comment by peleghila** [comment]
Actually `--lang sygus1` looks like it's working just fine. Anyway, it would be nice to remove this from the `--help` list just to avoid the confusion.

**Comment by nafur** [comment]
Ah, I see, my bad. Indeed, these changes came after the release. Though `sygus1` still works in the current release, I'd thus still suggest to move to `sygus2` at some point.

**Comment by peleghila** [comment]
It looks from the doc that `sygus` is supposed to indicate v2 (that's what I tried to use it for).

```
> cvc4-1.8-win64-opt.exe --lang help
Languages currently supported as arguments to the -L / --lang option:
  auto                           attempt to automatically determine language
  cvc4 | presentation | pl       CVC4 presentation language
  smt | smtlib | smt2 |
  smt2.0 | smtlib2 | smtlib2.0   SMT-LIB format 2.0
  smt2.5 | smtlib2.5             SMT-LIB format 2.5
  smt2.6 | smtlib2.6             SMT-LIB format 2.6 with support for the strings standard
  tptp                           TPTP format (cnf, fof and tff)
  sygus1                         SyGuS version 1.0
  sygus | sygus2                 SyGuS version 2.0
```

**Comment by nafur** [comment]
This issue was fixed with #4884. Please come back to use if you have any further problems.
